conf/prometheus-near.yml

```diff
@@ -0,0 +1,21 @@
+# my global config
+global:
+  scrape_interval:     5s
+  evaluation_interval: 5s
+  scrape_timeout: 2s
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+  # - "first_rules.yml"
+  # - "second_rules.yml"
+
+# A scrape configuration containing exactly one endpoint to scrape:
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'nearcore'
+
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+
+    static_configs:
+    - targets: ['localhost:3030']
```

Comment:
I think add more host here we could reuse one standalone prometheus instance to collect several near nodes.